specs/eventing/broker.md

```diff
@@ -145,6 +145,20 @@ The subscriber MAY receive a confirmation that a reply event was accepted by the
 Broker. If the reply event was not accepted, the initial event SHOULD be
 redelivered to the subscriber.
 
+Subscribers SHOULD respond to event delivery with one of the following HTTP
```

Comment:
We have not been this strict on the exact response code, accepting `2xx` means ACK, but we could have a MAY section that says here is what a `204` and a `200` could mean.

Reply:
Hi @n3wscott - thanks for the feedback. I have made some adjustments to the text of the PR. PTAL when you can.